src/MongoDB.Driver.Core/Core/Authentication/IAuthenticatorsFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+
+namespace MongoDB.Driver.Core.Authentication
+{
+    /// <summary>
+    /// Represents an authenticators factory.
+    /// </summary>
+    public interface IAuthenticatorsFactory
+    {
+        /// <summary>
+        /// Create authenticators.
+        /// </summary>
+        /// <returns>The authenticators.</returns>
+        IReadOnlyList<IAuthenticator> Create();
```

Comment:
Replace with:

    /// <summary>
    /// Create an authenticator.
    /// </summary>
    /// <returns>The authenticator.</returns>
    IAuthenticator Create();